README.md

```diff
@@ -14,18 +14,48 @@ files for assignments that require GatorGradle. ConfiGator uses
 
 ## Usage
 
-### Installing Python dependencies
+### Installing ConfiGator
 
-In order to install the dependencies of this project, run this command in your
-command line:
+Command to install ConfiGator
 
 ```bash
-poetry install
+pipx install configator
 ```
 
-After installing the dependencies, navigate to the repository directory, and run
-the following in your command line:
+### Commands to Change Specific Information Within a File
+
+Command to change name
 
 ```bash
-poetry run python main.py
+configator --help
 ```
+
+This command will show a list of every available command to change specific information
+within a file of your choice. The list will include commands to change the
+name of the assignment, change if gradle run breaks if there is a check failure,
+change if gradle run should immediately break upon a check failure through fastfail,
+change the indentation the body of the file should use, and the GatorGrader version
+of the assignment.
+
+### Testing Program
+
+Command to test the coverage
+
+```bash
+poetry run task test
+```
```

Comment:
This command, which would only be run by contributors, doesn't need to be included if you're not including instructions for contributing.